app/models/StripeAccountSetup.class.js

```diff
@@ -5,6 +5,8 @@ class StripeAccountSetup {
     this.bankAccount = opts.bank_account
     this.vatNumberCompanyNumber = opts.vat_number_company_number
```

Comment:
I think uses of this have been removed so it can probably be removed from the model

Reply:
👍 this is going into separate PR to remove here and all references